### ISSUE TITLE
录制帧率问题

### ISSUE THREAD

**Issue by IEtheReaLKnifL** [NickeManarin/ScreenToGif#405]
软件最高只支持60FPS, 我是120HZ的显示器, 录制的时候特别卡, 而且在全屏录制的时候CPU占用几乎100%
但是最骚的是录制的GIF很丝滑, 没有录制时的卡顿

**Comment by vatterspun** [comment]
> Care of Google Translate: The software only supports 60FPS, I am a 120HZ monitor, special card when recording, and the CPU is almost 100% when recording in full screen.
But the most confusing thing is that the recorded GIF is very silky, there is no recording when it is stuck.

Do you need to record above 60FPS?  Or are you concerned about the high CPU usage?

**Comment by IEtheReaLKnifL** [comment]
No, I don't need to record a 120FPS GIF.
My issue is when I record 60FPS GIF in full screen, the mouse slide is very stuck, and the CPU usage is very high, and my CPU is 7700

![image](https://user-images.githubusercontent.com/28704261/50053856-9e975f80-0176-11e9-8bed-4d6b0a32d57e.png)

**Comment by NickeManarin** [comment]
> My issue is when I record 60FPS GIF in full screen, the mouse slide is very stuck, and the CPU usage is very high, my CPU is 7700

Indeed, the recording is a heavy operation. In order to be less resource demanding, I'll need to use DirectX.
